src/components/section-editorial/section-editorial.js

```diff
@@ -123,7 +130,7 @@ const SectionEditorial = ({
                   components.map((item,index) => {                    
                     const Switcher = SwitchComponents[item.name]
                     return(
-                      <Switcher key={"switcher-"+index} {...item}/>
+                      <Switcher key={"switcher-"+index} {...item} componentViewerData={componentViewerData}/>
```

Comment:
I'm not familiar with the components in `SwitchComponents`, but I guess `componentViewerData` is specific to only a few of them and not all.

I think it would be more readable and future proof to pass `componentViewerData` only to the components that support it.